process_sample.smk

```diff
@@ -14,7 +14,7 @@ def check_for_ml_tag(bams):
     for bam in bams:
         with pysam.AlignmentFile(bam, "rb", check_sq=False) as bamfile:
             for record in bamfile:
-                if record.has_tag("Ml"):
+                if record.has_tag("Ml") or record.has_tag("ML"):
```

Comment:
Please revert this change.

Reply:
@williamrowell Aaron made that commit. I told him about the ML tag issue, but we didn't notice you already had another pull request to detect from header instead.

Reply:
Reverted the `ML` tag change in `main`. We don't commit code changes directly to main.